modules/tinymce/src/core/test/ts/browser/selection/SetSelectionContentTest.ts

```diff
@@ -242,6 +242,48 @@ UnitTest.asynctest('browser.tinymce.selection.SetSelectionContentTest', function
           })),
           root
         )
+      ]),
+
+      Log.stepsAsStep('TINY-5966', 'Set text content into empty pre block with leading/trailing spaces', [
+        tinyApis.sSetContent('<pre></pre>'),
+        tinyApis.sSetSelection([ 0, 0 ], 0, [ 0, 0 ], 0),
```

Comment:
```suggestion
        tinyApis.sSetCursor([ 0, 0 ], 0),
```

Just a nitpick. Take it or leave it.

Reply:
Fixed